src/riak_cs_gc_key_list.erl

```diff
@@ -180,6 +182,46 @@ gc_index_query(RcPid, StartKey, EndKey, BatchSize, Continuation, UsePaginatedInd
 
     {QueryResult, {StartKey, EndKey}}.
 
+-spec find_oldest_entries(BagId::binary()|master) ->
+                                 {ok, [{string(), [char()]}]} |
```

Comment:
This number should be same as the value made in `random:unidorm/1` in `riak_cs_gc:key_suffix/1`.